README.md

```diff
@@ -1 +1,63 @@
-# FunnySectors
\ No newline at end of file
+
+# Funny Sectors
+
+A lightweight plugin for managing large minecraft networks
+
+
+## Important note
+
+We do not have a working demo yet. [we](https://github.com/FunnyGuilds/FunnySectors/graphs/contributors) are actively working on creating one.
+
+
+## FAQ
+
+#### What are the priorities for the project?
+
+Well... We need a working demo. That is the most important thing for now. As [bopke](https://github.com/bopke) (project leader) [mentioned](https://github.com/FunnyGuilds/FunnySectors/issues/6) we need a working demo to avoid burnout.
+
+#### What technology are you going to start with?
+
+* Project language: [Java](https://www.java.com/en/)
+* Test language: [Kotlin](https://kotlinlang.org/)
+* Database: [MySql](https://www.mysql.com/)
+* Message broker: [Redis](https://redis.io/)
+* Minecraft platform: [Spigot](https://www.spigotmc.org/)
+* Proxy: [Velocity](https://velocitypowered.com/)
+
+All the technologies have been discussed [here](https://github.com/FunnyGuilds/FunnySectors/issues)
+
+### Are you planing on adding support for XYZ?
+
+We are planing to have a good abstraction layer so adding anything is not that hard. If you really care you can create issue or pull request. All contributions are welcome!
+
+### Is there any way to contact you?
+
+Yes! You can join project discord [here](https://discord.gg/CYvyq3u)
+
+
+
+
+## Roadmap
+
+- create project (setup [modules](https://github.com/FunnyGuilds/FunnySectors/issues/9))
+
+- create abstraction layer for message broker, database
+
+- create spigot "one source of truth" (Database as a way of storing players equipment)
+
+- split map into small areas (chunks) and add a way to send player between them
+
+- send player equipment using message broker and recive it on the other end
+
+- add some way to sync map between sectors
+
+- optymalize everythin
+
+- release first demo version
+
+## Logo
+
+Well... We do not have one. You are more then welcome to create one :)
```

Comment:
Why is this section a thing, do a project at this moment really need logo? What for?

Reply:
As I explained in [this](https://github.com/FunnyGuilds/FunnySectors/pull/10#discussion_r908354627) comment PR is important. Having a nice readme will attract contributors and server admins to the project. That is why having a logo is essential. We might not need it not but in a couple of months when we release first stable version we will. More project users, contributors = better. We accomplish that by having a nice looking readme and having a logo provides one.

Reply:
😖 so be it